backend/database/factories/UserFactory.php

```diff
@@ -15,11 +15,19 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'name' => $this->faker->name(),
+            'id' => $this->faker->unique()->uuid(),
+            'user_name' => $this->faker->userName(),
+            // password attribute is mutated when being created, see backend\app\Models\User.php
+            // 'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password <-- this is default from app creation
+            'password' => 'password',
+            'is_admin' => $this->faker->boolean(),
+            'first_name' => $this->faker->firstName(),
+            'last_name' => $this->faker->lastName(),
+            'middle_name' => $this->faker->lastName(),
             'email' => $this->faker->unique()->safeEmail(),
-            'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-            'remember_token' => Str::random(10),
+            'address' => $this->faker->unique()->address(),
+            'birthday' => $this->faker->dateTime(),
+            // 'remember_token' => Str::random(10),
```

Comment:
remove if not needed

```suggestion
            // 'remember_token' => Str::random(10),
```